pkg/eventservice/event_broker.go

```diff
@@ -464,6 +488,8 @@ func (c *eventBroker) doScan(ctx context.Context, task scanTask) {
 		c.emitSyncPointEventIfNeeded(dml.CommitTs, task, remoteID)
 		c.getMessageCh(task.workerIndex) <- newWrapDMLEvent(remoteID, dml, task.getEventSenderState())
 		metricEventServiceSendKvCount.Add(float64(dml.Len()))
+		lastSentDMLCommitTs = dml.CommitTs
```

Comment:
`lastSentDMLCommitTs` is a local variable, and not returned, so there is no need to set it to the `dml.CommitTs` ?